index.bs

```diff
@@ -7652,6 +7652,121 @@ To <dfn abstract-op>Create a new supplemental public key record</dfn>, perform t
         [=set/append=] this [=supplemental public key record=] to |credentialRecord|.[$credential record/supplementalPubKeys$].
 
 
+### Simple Confirmation Extension (confirmation) ### {#sctn-simple-txauth-extension}
```

Comment:
Do we still want the link to this section to reference txAuthExtension?

Reply:
addressed